src/main/scala/cromwell/binding/types/WdlType.scala

```diff
@@ -29,3 +29,14 @@ trait WdlType {
 
   def toWdlString: String
 }
+
+object WdlType {
+
+  private lazy val wdlTypes = Seq(
```

Comment:
I prefer using `Vector` as the concrete instantiation of `Seq` unless there's a reason not to